cli/flox-rust-sdk/src/flox.rs

```diff
@@ -33,7 +33,9 @@ use crate::providers::git::GitProvider;
 
 static INPUT_CHARS: Lazy<Vec<char>> = Lazy::new(|| ('a'..='t').collect());
 
-pub const FLOX_VERSION: &str = env!("FLOX_VERSION");
+pub fn get_flox_version() -> String {
+    std::env::var("FLOX_VERSION").unwrap_or(env!("FLOX_VERSION").to_string())
+}
```

Comment:
We've been using lazy's like this in other places to do runtime overridable inputs

```suggestion
pub static FLOX_VERSION: Lazy<String> = Lazy::new(||
    std::env::var("FLOX_VERSION").unwrap_or(env!("FLOX_VERSION").to_string())
);
```